app/processing/is-agreement-closed.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../data')
+
+const isAgreementClosed = async (paymentRequest) => {
+  const schemeId = paymentRequest.schemeId
+  const frn = paymentRequest.frn
+  const agreementNumber = paymentRequest.agreementNumber
```

Comment:
You could simplify this to `const { schemeId, frn, agreementNumber } = paymentRequest`

(admit this is a bit nitpicky, happy to go with whatever you think)

Reply:
simplified as per suggestion - much neater